core-java-modules/core-java-optional/src/test/java/com/baeldung/optionaluses/UsesForOptionalUnitTest.java

```diff
@@ -0,0 +1,64 @@
+package com.baeldung.optionaluses;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class UsesForOptionalUnitTest {
+
+    @Test
+    public void givenNonExistentUserId_whenSearchForUser_andNoNullCheck_thenThrowException() {
```

Comment:
junit 5 the `public` is optional

```suggestion
    void givenNonExistentUserId_whenSearchForUser_andNoNullCheck_thenThrowException() {
```